### PR TITLE
Papercut

### PR DESCRIPTION
**Pull request by jdhorwitz**
r? @steveklabnik

### CODE REVIEW

**Comment by steveklabnik** [comment]
Hey @jdhorwitz ! This looks like a start, but you have the commit from your other PR in here as well. Ping me on IRC if you need help sorting that out!

Also, `rustfmt` has betrayed you again 😄

File: src/libstd/process.rs [hunk 1]
```diff
@@ -343,6 +359,23 @@ impl Command {
 
     /// Add an argument to pass to the program.
     ///
+    /// Only one argument can be passed per use.
+    /// So instead of:
```
Comment (steveklabnik) [inline src/libstd/process.rs]: this shouldn't be on a newline, just put it after the previous sentence

File: src/libstd/process.rs [hunk 2]
```diff
@@ -343,6 +359,23 @@ impl Command {
...
+    ///
+    /// ```
+    /// .arg("-C /path/to/repo")
+    /// ```
+    /// usage would be:
```
Comment (steveklabnik) [inline src/libstd/process.rs]: an extra `///` above this would be good, so there's space below just like the space above

**Review by steveklabnik — CHANGES_REQUESTED** [review]

File: src/libstd/process.rs (hunk 1 above)
Comment (jdhorwitz) [inline src/libstd/process.rs]: Good catch, fixed!

File: src/libstd/process.rs (hunk 2 above)
Comment (jdhorwitz) [inline src/libstd/process.rs]: Fixed!

**Review by steveklabnik — APPROVED** [review]

**Comment by steveklabnik** [comment]
@bors: r-

whoops, travis failure is legit for once.

@jdhorwitz you need to use

    ```ignore

on those examples, as they don't compile on their own

**Comment by jdhorwitz** [comment]
Fixed!, Thanks @steveklabnik

**Comment by steveklabnik** [comment]
Looks great! Thanks again 😄 

@bors: r+ rollup

**Comment by pooyapeyravi** [comment]
hi
thanks for help

**Comment by pooyapeyravi** [comment]
blockquote, div.yahoo_quoted { margin-left: 0 !important; border-left:1px #715FFA solid !important; padding-left:1ex !important; background-color:white !important; } 


Gmail p

On پنجشنبه, مارس 9, 2017, 14:56, Ariel Ben-Yehuda <notifications@github.com> wrote:


@bors retry

**Pull request merged**